### PR TITLE
Rotational Cipher : Changes made | Add to track

### PR DESCRIPTION
**Pull request by LeonDsouza**
Made all changes as asked by @FridaTveit in the previous pull request.

---

Reviewer Resources:

[Track Policies](https://github.com/exercism/xjava/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/rotational-cipher/src/example/java/RotationalCipher.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+public class RotationalCipher {
+
+    public String rotate(String data, int shiftKey) {
+        StringBuilder dataStringBuilder = new StringBuilder();
+        for (char c : data.toCharArray()) {
+            if (Character.isUpperCase(c)) {
```
Comment (FridaTveit) [inline exercises/rotational-cipher/src/example/java/RotationalCipher.java]: It looks much neater now with the `Character` methods 👍

File: exercises/rotational-cipher/src/example/java/RotationalCipher.java [hunk 2]
```diff
@@ -0,0 +1,38 @@
+
+public class RotationalCipher {
```
Comment (FridaTveit) [inline exercises/rotational-cipher/src/example/java/RotationalCipher.java]: Maybe the `shiftKey` could be provided in the constructor and stored as a field?

File: exercises/rotational-cipher/src/example/java/RotationalCipher.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+     */
+    public char replacementCharacter(char characterToReplace,
+            char alphabetCaseStart, char alphabetCaseEnd, int shiftKey) {
+        char replacementCharacter = (char) (characterToReplace + shiftKey);
+        if (replacementCharacter > alphabetCaseEnd) {
+            replacementCharacter = (char) ((alphabetCaseStart - 1) +
+             (replacementCharacter % alphabetCaseEnd));
+        }
+        return replacementCharacter;
+    }
```
Comment (chuckwondo) [inline exercises/rotational-cipher/src/example/java/RotationalCipher.java]: I believe the following would work:

```java
return ((characterToReplace - alphabetCaseStart) + shiftKey)
        % (alphabetCaseEnd - alphabetCaseStart);
```

**Comment by stkent** [comment]
This exercise is almost there!

@LeonDsouza do you have time to apply the requested polish and complete this addition? If so, let us know how we can help you. If not, that's fine; please comment to that effect, and one of the track maintainers will adopt this PR and complete the last remaining steps.

Thanks! ❤️

**Comment by stkent** [comment]
I spy a `.class` file in here that we should remove; this repo only contains source files :)

**Pull request closed without merging**